public/static/docs/api-reference/open.md

```diff
@@ -45,8 +45,8 @@ file can be tracked by DVC or by Git.
 This function makes a direct connection to the
 [remote storage](/doc/command-reference/remote/add#supported-storage-types)
 (except for Google Drive), so the file contents can be streamed as they are
-read. This means it does not require space on the disc to save the file before
-making it accessible. The only exception is when using Google Drive as
+read. This means it does not require disc space to save the file before making
```

Comment:
are read -> is read? (similar to data, it's strange to see `contents are`)

Reply:
"file contents are" seems correct to me. I can change it to "file content is" though. Same meaning